avm/res/network/virtual-network-gateway/main.bicep

```
@@ -310,8 +303,8 @@ module publicIPAddress 'br/public:avm/res/network/public-ip-address:0.2.0' = [
       publicIPAllocationMethod: gatewayPipAllocationMethod
       publicIpPrefixResourceId: !empty(publicIPPrefixResourceId) ? publicIPPrefixResourceId : ''
       tags: tags
-      skuName: 'Standard'
-      zones: contains(zoneRedundantSkus, skuName) ? publicIpZones : []
+      skuName: skuName == 'Basic' ? 'Basic' : 'Standard'
+      zones: skuName == 'Basic' ? [] : zones
```

```suggestion
      zones: skuName != 'Basic' ? zones : []
```
Just makes it a bit easier to read.